index.html

```diff
@@ -1,226 +1,174 @@
+        </p>
+        <ul class="tags">
+          <li>html</li>
+          <li>css</li>
+          <li>javaScript</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    </div>
+  </section>
+  <section class="about_myself">
+    <div class="about_me_intro">
+      <h2>About <br>Myself</h2>
+      <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work 
+        and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <a href="https://github.com/naanmohammed">LET'S CONNECT</a>
+      <ul>
+        <li><a href="www.twitter.com/_naanahmed" target="_blank"><img src="images/twitter.jpg"></a></li>
+        <li><a href="www.linkedin.com/in/naanahmed" target="_blank"><img src="images/i.jpg"><img src ="images/n.jpg"></a></li>
+        <li><a href="www.medium.com/naanahmed" target="_blank"><img src="images/medium.jpg"></a></li>
+        <li><a href="www.github.com/naanmohammed" target="_blank"><img src="images/github.jpg"></a></li>
+        <li><a href="www.reddit.com/naanahmed" target="_blank"><img src="images/reddit.jpg"></a></li>
+      </ul>
+      <button>Get my resume</button>
     </div>
-    <section class="purple">
-    <div class="contact_me">
-        <h2>Contact me</h2>
-        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-
-        <form action="https://formspree.io/f/xvonabyg"
-        method="POST">
-            <ul>
-                <li><input value="" placeholder="Enter your name" id="name" type="text" name="user_name" maxlength="30" required></li>
-                <li><input value="" placeholder="Enter your email" id="email" type="email" name="email" required></li>
-            </ul>
-            <textarea value="" id="message" placeholder="Write your message here" type="text" name="message" maxlength="500" required></textarea>
-            <button>Get in touch</button>
-        </form>
+    <div class="languages">
+      <div class="language_header">
+        <h3>Languages</h3>
+        <p class="down_arrow"></p>
+      </div>
+      <ul class="language_list">
+        <li><img src="images/javascript.jpg"><p>Javascript</p></li>
+        <li><img src="images/html.jpg"><p>HTML</p></li>
+        <li><img src="images/css.jpg"><p>CSS</p></li>
+      </ul>
+      <div class="skill_frameworks">
+        <h3>Frameworks</h3>
+        <p class="icon_right"></p>
+      </div>
+      <div class="skill_frameworks">
+        <h3>Skills</h3>
+        <p class="icon_right"></p>
+      </div>
     </div>
-    </section>
+  </section>
+  <section class="contact_me">
+    <h2>Contact me</h2>
+    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    <form action="https://formspree.io/f/xvonabyg">
```

Comment:
The form is currently not submitted because you have not included the `method` attribute. I suggest adding `method="POST" as an attribute to the form in order to make the form submit to formspree

HERE IS A SCREENSHOT

![spree](https://user-images.githubusercontent.com/84930355/222307422-92b46161-094e-48fb-90b6-5c3783c49fa7.PNG)



Furthermore, the form being the parent of the input elements should align its children using flex/Doing so makes styling much easier and more flexible.